setup.cfg

```diff
@@ -21,8 +21,6 @@ description = Command line client for interaction with DANDI archive elements
 long_description = file:README.md
 long_description_content_type = text/markdown; charset=UTF-8
 platforms = OS Independent
-provides =
-    dandi
```

Comment:
just curious: where it potentially could be used? I found no description on https://packaging.python.org/guides/distributing-packages-using-setuptools

Reply:
Nothing in the ecosystem currently uses this field.  There are occasional discussions around using "Provides" to implement support for things like virtual packages, but nothing ever comes of it.